### PR TITLE
added model building part and uni-variate analysis of the target variable

### PR DESCRIPTION
**Pull request by soumya997**
Added one .ipynb, .py and a requirement.txt file.
PR for issue #4 .
Made a model using Random Forest Regressor which is giving an r2 score of 0.04, did a Uni-variate analysis of the target vatriavle(area), and added a requirement.txt file.

### CODE REVIEW

**Comment by soumya997** [comment]
hey @Nibba2018 ,I used cufflinks and plotly for making the plots but it seems like the plots are not visible.
This is how it really looks like --> https://www.kaggle.com/soumya9977/forest-fire-prediction-draft-2
What should I do, make plots using matplotlib, and seaborn?
can you check the notebook by downloading it?

**Comment by BALaka-18** [comment]
> hey @Nibba2018 ,I used cufflinks and plotly for making the plots but it seems like the plots are not visible.
> This is how it really looks like --> https://www.kaggle.com/soumya9977/forest-fire-prediction-draft-2
> What should I do, make plots using matplotlib, and seaborn?
> can you check the notebook by downloading it?

@soumya997 cufflinks and plotly plots are not visible in GitHub. You can do one thing, write an introductory note at the beginning of the notebook, notifying the user to download the notebook for viewing all plots.

File: submissions/PythonScripts/0.04_random_Forest_Regressor_Soumyadip.py [hunk 1]
```diff
@@ -0,0 +1,587 @@
...
+#       * Encoding of data
+#       * Feature selection
+#       * Train test split
+# <br>     
+# 5. Model building
+#       * Model Building using Selected features
+#       * Cross Validation
+#       * Metric Reports
+#       
+#
```
Comment (BALaka-18) [inline submissions/PythonScripts/0.04_random_Forest_Regressor_Soumyadip.py:38]: Add the Introductory note as I have mentioned.

**Review by BALaka-18 — CHANGES_REQUESTED** [review]

File: submissions/PythonScripts/0.04_random_Forest_Regressor_Soumyadip.py (hunk 1 above)
Comment (soumya997) [reply submissions/PythonScripts/0.04_random_Forest_Regressor_Soumyadip.py:38]: Ok, great.
So, all I have to do is put an Introductory note at the bottom of the table of content (in the line number 39 to 40), right? @BALaka-18

Comment (BALaka-18) [reply submissions/PythonScripts/0.04_random_Forest_Regressor_Soumyadip.py:38]: Yes.

Comment (soumya997) [reply submissions/PythonScripts/0.04_random_Forest_Regressor_Soumyadip.py:38]: @BALaka-18 please check now.

Comment (soumya997) [reply submissions/PythonScripts/0.04_random_Forest_Regressor_Soumyadip.py:38]: Hey, @BALaka-18 should I change the plots to matplotlib and seaborn, as I am going to do the multivariate analysis part too I guess it won't take much time to do.

But the original notebook looks really cool i feel, just not in gihub.

Comment (BALaka-18) [reply submissions/PythonScripts/0.04_random_Forest_Regressor_Soumyadip.py:38]: @soumya997 let it stay, those libraries are also important. And anyways people can always download your notebook and view them whenever they want.

**Review by BALaka-18 — APPROVED** [review]
Lgtm. Ready to merge.

**Comment by BALaka-18** [comment]
Hey @soumya997 , I hope you generated the ```requirements.txt``` file inside a virtual environment ?

**Comment by soumya997** [comment]
oh I did not see that, and yes

> Hey @soumya997 , I hope you generated the `requirements.txt` file inside a virtual environment ?

**Pull request merged**